### PR TITLE
Refine mission statement wording

### PR DESCRIPTION
**Pull request by dthaler**
Editorial cleanup for grammar and terminology consistency

### CODE REVIEW

**Review by lkatalin — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -13,12 +13,12 @@ Confidential Computing is the protection of data in use by performing computatio
...
-* defining confidential computing and accelerate acceptance and adoption in the
+* defining confidential computing and accelerating acceptance and adoption in the
 market.
-*  developing open enterprise-grade building blocks (e.g. open specifications and open
-source licensed projects), including a confidential managed compute portfolio and
-development experience, to enable easy development and management of enterprisegrade confidential compute applications, and
-* defining foundational services and frameworks that are confidential-aware and
+* developing open enterprise-grade building blocks (e.g., open specifications and open
+source licensed projects), including a confidential computing portfolio and
+development experience, to enable easy development and management of enterprise-grade confidential computing applications, and
```
Comment (MikeCamel) [inline README.md:20]: I'd like to see us moving more to Confidential Computing (capitali[s|z]ed) everywhere (and in line 6 above), by preference.  Thoughts?

File: README.md [hunk 2]
```diff
@@ -13,12 +13,12 @@ Confidential Computing is the protection of data in use by performing computatio
...
+* defining foundational services and frameworks that are confidential-computing aware and
 minimize the need for trust.
```
Comment (MikeCamel) [inline README.md:23]: Add something around "raising industry awareness of Confidential Computing and use cases across various sectors"?  Also, maybe regulators and standards body engagement?  If so, this stuff needs to go to the GB.

File: README.md [hunk 3]
```diff
@@ -13,12 +13,12 @@ Confidential Computing is the protection of data in use by performing computatio
...
 relating to:
-* defining confidential computing and accelerate acceptance and adoption in the
+* defining confidential computing and accelerating acceptance and adoption in the
 market.
-*  developing open enterprise-grade building blocks (e.g. open specifications and open
-source licensed projects), including a confidential managed compute portfolio and
-development experience, to enable easy development and management of enterprisegrade confidential compute applications, and
-* defining foundational services and frameworks that are confidential-aware and
+* developing open enterprise-grade building blocks (e.g., open specifications and open
+source licensed projects), including a confidential computing portfolio and
```
Comment (dcmiddle) [inline README.md:19]:
```suggestion
source licensed projects), including a Confidential Computing portfolio and
```

File: README.md [hunk 4]
```diff
@@ -13,12 +13,12 @@ Confidential Computing is the protection of data in use by performing computatio
...
+development experience, to enable easy development and management of enterprise-grade confidential computing applications, and
+* defining foundational services and frameworks that are confidential-computing aware and
```
Comment (dcmiddle) [inline README.md:21]:
```suggestion
* defining foundational services and frameworks that are Confidential Computing aware and
```

File: README.md (hunk 2 above)
Comment (dcmiddle) [reply README.md:23]: This text comes from the charter (grammatical errors and all). I'm fine cleaning it up here but I agree that any substantive rewrite is a board action outside the scope of this PR.

File: README.md (hunk 1 above)
Comment (dthaler) [reply README.md:20]: Sounds good to me.

File: README.md [hunk 5]
```diff
@@ -13,12 +13,12 @@ Confidential Computing is the protection of data in use by performing computatio
...
-* defining confidential computing and accelerate acceptance and adoption in the
+* defining confidential computing and accelerating acceptance and adoption in the
 market.
-*  developing open enterprise-grade building blocks (e.g. open specifications and open
-source licensed projects), including a confidential managed compute portfolio and
-development experience, to enable easy development and management of enterprisegrade confidential compute applications, and
-* defining foundational services and frameworks that are confidential-aware and
+* developing open enterprise-grade building blocks (e.g., open specifications and open
+source licensed projects), including a Confidential Computing portfolio and
+development experience, to enable easy development and management of enterprise-grade confidential computing applications, and
```
Comment (dthaler) [inline README.md:20]:
```suggestion
development experience, to enable easy development and management of enterprise-grade Confidential Computing applications, and
```

File: README.md (hunk 2 above)
Comment (dthaler) [reply README.md:23]: Yeah, https://confidentialcomputing.io/wp-content/uploads/sites/10/2023/03/Confidential-Computing-Consortium-Charter.pdf Exhibit B, point 1(a) has the same issues.  Dan can you raise this PR with the board as a charter amendment to vote on?

**Review by cfircohen — APPROVED** [review]

**Review by dcmiddle — APPROVED** [review]

**Pull request merged**